src/docs/Chakra.mdx

```diff
@@ -473,118 +203,3 @@ props which are shorthand for "padding" and "max-width", respectively.
 
 Find more information and a full list of all the available props, check the Chakra
 UI [Style Props page](https://chakra-ui.com/docs/styled-system/features/style-props).
-
-## Hooks
-
-The following hooks are available to use and can be imported from
-`@nypl/design-system-react-components`.
-
-### useNYPLTheme
```

Comment:
I think this is still useful to have here. But, the specific docs for this hook should have the same, if not better, docs. I also don't know anyone who uses this so I think it's ok to remove.

Reply:
I didn't think it belonged in a doc called, "Chakra", but I may be missing something!